hardware/OnkyoAVTCP.h

```diff
@@ -3,6 +3,7 @@
 #include <iosfwd>
 #include "ASyncTCP.h"
 #include "DomoticzHardware.h"
+#include <map>
```

Comment:
I see here a map include but no reference to a map usage in this header? If it is used in the .cpp, then please include this in the .cpp only

Reply:
Yep, there *was* a map but I simplified things. Will fix it.